backend/app/cmd/server.go

```diff
@@ -912,11 +914,11 @@ func (s *ServerCommand) loadEmailTemplate() (string, error) {
 	return string(file), nil
 }
 
-// aside from notify.Service and error, returns telegram bot name which will be passed to the frontend
-func (s *ServerCommand) makeNotify(dataStore *service.DataStore, authenticator *auth.Service) (*notify.Service, string, error) {
+// aside from notify.Service and error, returns telegram service
+func (s *ServerCommand) makeNotify(dataStore *service.DataStore, authenticator *auth.Service) (*notify.Service, *notify.Telegram, error) {
```

Comment:
if tg needs a separate return value, maybe make it a separate func? like makeTgNotify?